Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'dalli'
 gem 'gds-api-adapters', '~> 43.0'
 gem 'govuk_ab_testing', '~> 2.0'
 gem 'govuk_frontend_toolkit', '5.1.0'
-gem 'govuk_publishing_components', '~> 1.5.0', require: false
+gem 'govuk_publishing_components', '~> 1.6.0', require: ENV['RAILS_ENV'] != "production" || ENV['HEROKU_APP_NAME'].to_s.length.positive?
```

Comment:
This isn't the way we require the gem now.

See:
https://github.com/alphagov/govuk_publishing_components/pull/56
https://github.com/alphagov/government-frontend/pull/478